### DISCUSSION TITLE
Welcome to DashPlayer Discussions!

### DISCUSSION THREAD

**Discussion by solidSpoon** [solidSpoon/DashPlayer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by huge689** [comment]
大佬,有个功能需求: 自动跳过字幕之间的时间间隙, 这样学习效率高很多.

**Reply by solidSpoon** [reply]
OK，这个想法挺好，感谢

**Comment by OlivierGao** [comment]
遇到一个问题：我在GPTAPI.US上买了openai的API，试了两个视频，生成字幕都是转录到40%停了。显示的状态是connection error。请问大佬是怎么回事？我人在海外，没在国内，不存在任何科学上网问题
